tests/conftest.py

```diff
@@ -96,19 +96,24 @@ def cleanup_test_databases(config: ConfigFile, postgres_db_engine: sa.Engine) ->
 def dbos(
     config: ConfigFile, cleanup_test_databases: None
 ) -> Generator[DBOS, Any, None]:
+    DBOS.clear_global_instance()
     dbos = DBOS(config=config)
+    dbos.launch()
```

Comment:
Why is this call needed?  Why is it not needed in the other fixture?

Reply:
I can put it in the other fixture, but tests like admin server don't trigger the launch in time otherwise.